### ISSUE TITLE
fp16 training loss=nan

### ISSUE THREAD

**Issue by ilaij0810** [ZJULearning/resa#48]
hi, thank you for your work！
I have encountered a problem when I set fp16 training loss is always nan.  then i found in **resa** module, after down, up, right and left feature fusion, the feature value become very large, and many values are larger than 65504, so the actually value becomes **inf**. How can I achieve mixed precision（fp16） training without losing too much performance?

**Comment by ilaij0810** [comment]
> 

I have trid some method, add BN to conv in resa module, 
![image](https://user-images.githubusercontent.com/42317829/202147212-e81135c0-5406-4385-b63b-702cb3818f92.png)
but no lanes detected.
if I decrease the value of alpha, i.e, alpha=0.1, or change the act (original is relu) to sigmoid or tanh, ..,  it will losing too much performance?

expecting your reply.

**Comment by 2696120622** [comment]
@ilaij0810 
I have the same problem of loss=nan. If I set alpha to 1.0, I have not got the loss of nan. But, the training can not coverge.
Do you have any solutions?
